src/components/TodoModal/TodoModal.tsx

```diff
@@ -1,39 +1,67 @@
-import React from 'react';
+import React, { SetStateAction, useEffect, useState } from 'react';
+import { getUser } from '../../api';
+import { Todo } from '../../types/Todo';
+import { User } from '../../types/User';
 import { Loader } from '../Loader';
 
-export const TodoModal: React.FC = () => {
+type Props = {
+  selectedTodo: Todo;
+  setSelectedTodo: React.Dispatch<SetStateAction<Todo | null>>;
```

Comment:
The name of the prop `setSelectedTodo` doesn't follow the recommended naming conventions for event handler functions. It would be better to name it `onTodoSelect` or `handleTodoSelect`.